web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/DownloadImageControllerIT.java

```diff
@@ -390,13 +394,4 @@ public void testGetAccessImageSolrUnavailable() throws Exception {
                 .andExpect(status().isServiceUnavailable())
                 .andReturn();
     }
-
-    private void assertCorrectImageReturned(MockHttpServletResponse response) throws IOException {
-        assertEquals("image/jpeg", response.getContentType());
-
-        var responseContent = response.getContentAsByteArray();
-        byte[] imageContent = FileUtils.readFileToByteArray(new File("src/test/resources/__files/bunny.jpg"));
```

Comment:
i hadn't noticed before, but why is the folder named "__files" rather than just "files"?

Reply:
no idea! i don't remember what happened when i made that directory

Reply:
oh turns out it's needed for the iiif v3 service to find it

Reply:
WHOOPS. it's needed for wiremock
```
Wiremock Documentation states that the location of the file specified in withBodyFile should be in src/test/resources/__files.
```